### ISSUE TITLE
Can we get support for images/create-edit & images/create-variation?

### ISSUE THREAD

**Issue by LawrenceNorman** [adamrushy/OpenAISwift#75]
Just noticed that these APIs were available:
https://platform.openai.com/docs/api-reference/images/create-edit
https://platform.openai.com/docs/api-reference/images/create-variation

Would be nice to have these added!

**Comment by optera15** [comment]
Agreed! It's somewhat odd how progress on this project is so seemingly sleepy given the enormity of the power of these OpenAI APIs and how easy it should be to enhance/expand this package through use of GPT-4 itself. I wrapped the salient parts of the ElevenLabs API in 20 minutes yesterday, or rather, GPT-4 did as per my instructions. 

Then again, like everyone else I'm too caught up in doing other development work with AI right now to contribute much of anything myself to OpenAISwift.

**Comment by MarkHoath** [comment]
I have just completed this and hopefully it gets merged quickly.

Because of the different handling of images between iOS and MacOS, you will need to do all the checking prior to sending paramters (dimensions, size, alpha channels).
